doc/1/controllers/index/refresh/index.md

```diff
@@ -0,0 +1,32 @@
+---
+code: true
+type: page
+title: RefreshAsync
+description: Force Elasticsearch search index update.
+---
+
+# RefreshAsync
+
+When writing or deleting documents in Kuzzle, the update needs to be indexed before being available in search results.
```

Comment:
It seems that the action description is missing (comparing to the [API equivalent](https://docs.kuzzle.io/core/1/api/controllers/index/refresh/))

Reply:
I have added `Forces an immediate [reindexation](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/docs-refresh.html) of the provided index.` above this line.